components/brave_wallet_ui/panel/container.tsx

```diff
@@ -208,20 +208,22 @@ function Container() {
 
   if (addChainRequest) {
     return (
-      <PanelWrapper isLonger={true}>
-        <LongWrapper>
-          <AllowAddChangeNetworkPanel addChainRequest={addChainRequest} />
-        </LongWrapper>
+      <PanelWrapper
+        width={390}
+        height={650}
+      >
```

Comment:
nit: What's the reason for fixing these sizes now rather than using longWrapper?